scripts/notify.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+## usage:
+# ./notify.py FAILURE
+# ./notify.py SUCCESS
+
+import os
+import json
+import sys
+import requests
+import datetime
+from logsnag import LogSnag
+import slack_notifications as slack
+
+
+#the 4 notification channels:
+# - slack
+# - notifi (works for both phone and mac)
+# - push on the phone (via 'logsnag', pushme disappeared)
+# - telegram (wip)
+
+
+token_file = f'{os.path.realpath(os.path.dirname(__file__))}/../secrets/tokens.json'
+
+def open_credentials_file():
+  with open(token_file) as tokens_json:
+    return json.load(tokens_json)
+
+tokens = open_credentials_file()
+
+
+STATUS = sys.argv[1]
+EVENT = f'The backup was a {STATUS}'
+
+def slack_alert():
+  api_key = tokens["slack"]
+
+  slack.ACCESS_TOKEN = api_key
+
+  attachment = slack.Attachment(
+      title='Backup result',
+      author_name=f'Host: {os.uname()[1]}',
+      text=EVENT,
+      footer='Hope the backup is never needed',
+      color='#37FDFC' if STATUS == "SUCCESS" else "#ff5a36",
+      fields=[
+        slack.Attachment.Field(
+            title='Completed at:',
+            value=f'{datetime.datetime.now().strftime("%d %b %Y, %H:%M:%S %Z")}',
+            short=False
+        ),
+      ],
+      #fallback=EVENT # missing
```

Comment:
Missing from where?

Reply:
the library doesn't provide that same parameter - idk what it meant though and I didn't want to check on slack API

Reply:
Let's remove it then

Reply:
Nvm, already done. Thanks!